sharepoint/sharepoint-ps/sharepoint-online/New-SPOSiteSharingReportJob

```diff
@@ -0,0 +1,97 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://docs.microsoft.com/powershell/module/sharepoint-online/New-SPOSiteSharingReportJob
+applicable: SharePoint Online
+title: New-SPOSiteSharingReportJob
+schema: 2.0.0
+author: salarson
+ms.author: salarson
+ms.reviewer:
+---
+
+# New-SPOSiteSharingReportJob
+
+## SYNOPSIS
+
+Creates a new sharing report job 
+
+## SYNTAX
+
+```powershell
+New-SPOSiteSharingReportJob -Site <SPOSitePipeBind> --ReportStorageWebUrl  <String> -ReportStorageFolderUrl <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet is not currently active in Production and may be removed in the future. You will receive the error, "Site collection sharing report feature has not been enabled", if you run this cmdlet which is currently by design. 
+
+## EXAMPLES
+
+### -----------------------EXAMPLE 1-----------------------------
+
+```powershell
+$site = Get-SPOSite -Identity https://contoso.sharepoint.com/sites/site1
+
+New-SPOSiteSharingReportJob -Site $site -ReportStorageWebUrl 'https://contoso.sharepoint.com/sites/site2/web1' -ReportStorageFolderUrl '/Documents/folder'
+```
+
+## PARAMETERS
+
+### -ReportStorageFolderUrl
+
+Location to where the report will be exported. 
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+Applicable: SharePoint Online
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ReportStorageWebUrl
+
+Report web storage URL 
+
+```yaml
+Type: String[]
```

Comment:
Is this truly a string array? If so, can you add a second example with an array and correct line 22 to show that it is `<string>[]`?

Reply:
This cmdlet wont be functional for 99.9% of tenants , I think this should suffice.

New-SPOSiteSharingReportJob -Site $site -ReportStorageWebUrl 'https://contoso.sharepoint.com/sites/site2/web1' -ReportStorageFolderUrl '/Documents/folder'